### ISSUE TITLE
Why there is a - in front of the regression_loss?

### ISSUE THREAD

**Issue by Wang-Yu-Qing** [google/lifetime_value#15]
in zero_inflated_lognormal.py line 76:
```
      regression_loss = -tf.keras.backend.mean(
      positive * tfd.LogNormal(loc=loc, scale=scale).log_prob(safe_labels),
      axis=-1)

     return classification_loss + regression_loss
```
In the paper, the Loss equals CrossEntropyLoss + LogNormalLoss, so why there is a minus in front of the LogNormalLoss?

**Comment by minh-nguyenhoang** [comment]
It's for negative log-likelihood (which you should see `.log_prob()`). In the normal regression setting, like MSE assume a normal distribution with known variance, the log-likelihood is `-(x-mu)^2 + constant`, which translate to the MSE loss as `- -(x-mu)^2`. Subtitute normal with laplace distribution, you have MAE loss.
